kalite/distributed/static/js/distributed/exercises/models.js

```diff
@@ -25,10 +29,12 @@ window.ExerciseDataModel = Backbone.Model.extend({
 
     initialize: function() {
 
-        _.bindAll(this);
+        _.bindAll(this, "url", "update_if_needed_then", "as_user_exercise", "get_framework");
+
+        var self = this;
 
         // store the provided seed as an object attribute, so it will be available after a fetch
-        this.listenTo(this, "change:seed", function() { this.seed = this.get("seed") || this.seed; });
+        this.listenTo(this, "change:seed", function() { self.seed = self.get("seed") || self.seed; });
```

Comment:
The scope of 'this' can become confused inside Javascript functions depending on the scope in which the function is called. In this case, I changed it, because this was referring to the wrong object.